compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -4252,6 +4258,15 @@ private BLangBindingPattern transformBindingPattern(Node bindingPattern) {
                             (BLangRestBindingPattern) transformBindingPattern(listMemberBindingPattern);
                 }
                 return bLangListBindingPattern;
+            case NAMED_ARG_BINDING_PATTERN:
```

Comment:
Lets move switch case logic that are too big to a separate method

Reply:
Fixed